ql/termstructures/volatility/optionlet/constantoptionletvol.hpp

```diff
@@ -35,29 +36,33 @@ namespace QuantLib {
     class ConstantOptionletVolatility : public OptionletVolatilityStructure {
       public:
         //! floating reference date, floating market data
-        ConstantOptionletVolatility(Natural settlementDays,
-                                    const Calendar& cal,
+        ConstantOptionletVolatility(Natural settlementDays, const Calendar &cal,
                                     BusinessDayConvention bdc,
-                                    const Handle<Quote>& volatility,
-                                    const DayCounter& dc);
+                                    const Handle< Quote > &volatility,
```

Comment:
Yes, that came up. Peter even sent his format file as an example. We can try to get a format file that's close to what we have in most files and add it to the repo. Contributions are welcome, of course :smile: